filters/openpolicyagent/openpolicyagent.go

```diff
@@ -292,9 +301,9 @@ func (registry *OpenPolicyAgentRegistry) newOpenPolicyAgentInstance(bundleName s
 		return nil, err
 	}
 
-	err = engine.waitPluginsReady(100*time.Millisecond, 30*time.Second)
+	err = engine.waitPluginsReady(100*time.Millisecond, config.startupTimeout)
```

Comment:
You will need to make this a default on config in case this is used without using the `WithStartupTimeout` option. All the options are optional an can be passed but it should work without making them mandatory

Reply:
Added in line 162